straxen/contexts.py

```diff
@@ -19,7 +19,7 @@
         straxen.PeakProximity],
     check_available=('raw_records', 'peak_basics'),
     store_run_fields=(
-        'name', 'number', 'tags.name',
```

Comment:
Is it necessary to remove this if we are fixing the rundb frontend to be able to handle it? or is this just outdated and being removed anyway?

Reply:
It's hardcoded here:
https://github.com/AxFoundation/strax/blob/master/strax/run_selection.py#L80

So we don't need this one, the correct info is stored under 'tags'